test/unit/Cardano/Wallet/PackfileSpec.hs

```diff
@@ -0,0 +1,56 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Cardano.Wallet.PackfileSpec (spec) where
+
+import Prelude
+
+import qualified Data.ByteString.Lazy.Char8 as L8
+
+import Test.Hspec
+    ( Spec, describe, it, shouldBe )
+
+import Cardano.Wallet.Binary.Packfile
+
+-- version 1 header
+packFileHeader :: L8.ByteString
+packFileHeader = "\254CARDANOPACK\NUL\NUL\NUL\SOH"
+
+test1 :: L8.ByteString
+test1 = packFileHeader <> "\0\0\0\5hello\0\0\0"
+
+test2 :: L8.ByteString
+test2 = packFileHeader
+        <> "\0\0\0\11first block\0"
+        <> "\0\0\0\12second block"
```

Comment:
Is there any benefit to writing these as top-level definitions? If so, then are we able to use more meaningful names?
(Feel free to ignore, just a suggestion.)

Reply:
Fixed